python/mdvtools/llm/langchain_mdv.py

```
@@ -0,0 +1,468 @@
+                     template=prompt_RAG,
+                     input_variables=["context", "question"])
+
+
+            with time_block("b12: RAG chain"):  # ~60% of time
+                self.socket_api.update_chat_progress(
+                    "Invoking RAG chain...", id, progress, 60
+                )
+                progress += 60
+
+                qa_chain = RetrievalQA.from_llm(llm=self.code_llm, prompt=prompt_RAG_template, retriever=retriever,return_source_documents=True,)
+                context = retriever
```

Ruff (0.11.9)

Local variable `context` is assigned to but never used

Remove assignment to unused variable `context`

(F841)